### PR TITLE
add after and before fields on contacts api endpoint to ilter on modi…

### PR DESCRIPTION
**Pull request by norkans7**
…fied_on

### CODE REVIEW

**Comment by nicpottier** [comment]
What about adding / removing URNs?

**Comment by norkans7** [comment]
Ohh right, I kept thinking in the context of ureport

File: temba/api/views.py [hunk 1]
```diff
@@ -1666,7 +1685,11 @@ def get_read_explorer(cls):
                           dict(name='urns', required=False,
                                help="One or more URNs to filter by.  ex: tel:+250788123123,twitter:ben"),
                           dict(name='group_uuids', required=False,
-                               help="One or more group UUIDs to filter by. (repeatable) ex: 6685e933-26e1-4363-a468-8f7268ab63a9")]
+                               help="One or more group UUIDs to filter by. (repeatable) ex: 6685e933-26e1-4363-a468-8f7268ab63a9"),
+                          dict(name='after', required=False,
+                                help="only contacts which have changed after this date.  ex: 2012-01-28T18:00:00.000"),
+                          dict(name='before', required=False,
+                                help="only contacts which have changed before this date. ex: 2012-01-28T18:00:00.000")]
```
Comment (nicpottier) [inline temba/api/views.py]: Might be good to have these imply that it is >= ie..

"only contacts which have changed on this date or before" etc..

File: temba/contacts/tests.py [hunk 2]
```diff
@@ -1753,6 +1757,14 @@ def test_update_handling(self):
         def create_dynamic_group(name, query):
             return ContactGroup.create(self.org, self.user, name, query=query)
 
+        self.bob = self.create_contact("Bob", "1234")
+        self.bob.name = 'Bob Marley'
+        self.bob.save()
+        old_modified_on = self.bob.modified_on
+        self.bob.handle_update()
+        self.assertTrue(self.bob.modified_on > old_modified_on)
+
```
Comment (nicpottier) [inline temba/contacts/tests.py]: Maybe add a URN test as well, that way we know it doesn't break in the future?

Comment (norkans7) [inline temba/contacts/tests.py]: I added a test for that now

**Comment by nicpottier** [comment]
Looks good!

**Pull request merged**